Paws4Me/Paws4Me/Controllers/Logic/SignInViewController.swift

```diff
@@ -11,20 +11,27 @@ protocol Validation {
 }
 
 class SignInViewController: UIViewController, Validation {
+    // MARK: - IBOulets
     @IBOutlet weak private var txtUsername: UITextField!
     @IBOutlet weak private var txtPassword: UITextField!
     @IBOutlet weak private var btnSignIn: UIButton!
     @IBOutlet weak private var btnRegister: UIButton!
```

Comment:
A tip for naming is starting with what the name and ending with the objects name. This will help speed up coding when using items. 

Example: 

registerButton
signinButton

This then stops massive auto complete when you type btn its going to give you a list of all the variables starting with btn 

Just a thought

Reply:
Thank you very much for this. I have already added this into my other branch